vscode-extensions/tls-sync-vscode-ui-extension/config/heft.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
We should probably just make a rig for vscode extensions that extends local-node-rig.

Reply:
Created `@rushstack/heft-vscode-extension-rig`